components/builder-http-gateway/src/http/middleware.rs

```diff
@@ -170,6 +171,7 @@ impl Key for Authenticated {
 
 impl BeforeMiddleware for Authenticated {
     fn before(&self, req: &mut Request) -> IronResult<()> {
+        aobafbp
```

Comment:
What's this?

Reply:
Unintentionally committed. I was in the middle of testing https://github.com/habitat-sh/builder/pull/109 and intentionally causing build failures